project/README.rst

```diff
@@ -65,3 +65,4 @@ Project List by Account Name
 
 #. `github_username <https://github.com/github_username>`__ for
    `Project title <github_username/README.rst>`__: https://github.com/github_username/project_name
+#. `Hooje <https://github.com/Hooje>`__ for Simple Parallel Decision Tree in C++ : https://github.com/Hooje/nsdhw_22au/tree/project/project/Hooje
```

Comment:
You should use a distinct repository for the project.

Reply:
Does that mean that this repository is just for proposal?

Reply:
Correct.  You should not use the homework repository for your code development.  There will be too much interference from the class.  It has a lot of administrative overheads to manage more than 20 contributors in a git repository.